src/other/PropertyEditor.js

```diff
@@ -92,7 +92,7 @@
     PropertyEditor.prototype.enter = function (domNode, element) {
         Accordion.prototype.enter.apply(this, arguments);
         
-        if(!this._parentElement.node().matches(".other_PropertyEditor div")){
+        if(this._parentElement.filter(".other_PropertyEditor div")[0].length === 0){
```

Comment:
I prefer the empty() method to the ```[0].length === 0``` as the in the latter you are breaking the object encapsulation (your assuming knowledge of the inner workings which may change in the future).